### ISSUE TITLE
Downloading jce_policy-6.zip failed

### ISSUE THREAD

**Issue by Boboss74** [flexiondotorg/oab-java6#103]
- Latest script version (6aa89d0f69c4dab039f32044865471db39fe9633)
- Ubuntu 12.10 x64

It tries to download jce_policy-6.zip from http://jce_policy-6.zip/

The logs are:

```
[x] Downloading jce_policy-6.zip :    --2013-06-20 12:08:03--  http://jce_policy-6.zip/
Resolving jce_policy-6.zip (jce_policy-6.zip)... failed: Name or service not known.
wget: unable to resolve host address `jce_policy-6.zip'
22069's retcode: 4
failed
```

**Comment by Boboss74** [comment]
Ops, duplicate issue with #101:Error running 0.2.8 tag
(not very explicit title u_u)

**Comment by flexiondotorg** [comment]
Merged pull-requuest #111.
